bcipy/tasks/task_registry.py

```diff
@@ -31,13 +31,13 @@ class ExperimentType(Enum):
     'Calibration'
     """
```

Comment:
This doctest should be updated as well. The returned value should be 'RSVP Calibration'. Also, should this enum be renamed to TaskType?

Reply:
I think that makes sense!